### ISSUE TITLE
❌ 🚧 Invalid Issue - Outdated: Fatal error when loading Elementor main dashboard

### ISSUE THREAD

**Issue by Dekadinious** [elementor/elementor#27697]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When loading /wp-admin/admin.php?page=elementor we get a fatal error.

```log
An error of type E_ERROR was caused in line 30 of the file ../elementor/modules/home/transformations/filter-plugins.php. Error message: Uncaught TypeError: Elementor\Modules\Home\Transformations\Filter_Plugins::get_add_ons_installation_status(): Argument #1 ($add_ons) must be of type array, null given, called in ../elementor/modules/home/transformations/filter-plugins.php on line 21 and defined in ../elementor/modules/home/transformations/filter-plugins.php:30
```

### Steps to reproduce

Load /wp-admin/admin.php?page=elementor

### Expected behavior

No fatal error.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

**[Moderation Edit: The contributor failed to share the required information outlined in the [Contribution Guidelines](https://elemn.to/gh-contributing-bug-reports).]**

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by fullstackdeveloper82** [comment]
Hi [Dekadinious]

Are you using any theme, if yes, could you please let me know the theme you are using, as I am not able to reproduce this issue.

As a temporary fix, you can downgrade your php version to 7.4

Thanks
Vikas

**Comment by Dekadinious** [comment]
Yes, we are using Astra Pro on this site and sadly am not able to downgrade the PHP-version. If able to do so securely, I could give you temporary admin access to the site if you want to have a closer look. @fullstackdeveloper82

**Comment by fullstackdeveloper82** [comment]
Sure, I'll need cpanel login details to downgrade php or fix this issue. vikas.kodesk@gmail.com

**Comment by Dekadinious** [comment]
We do not have cPanel and we will not downgrade PHP :) I was more thinking if you need any insights to why it might be happening, but I guess I can try a full stack trace when I have the time.

**Comment by bugnumber9** [comment]
I see the same error on PHP 7.4 using Hello theme.

**Comment by hernands** [comment]
Did anyone find a fix for this issue? I'm running into the same problem. Tried downgrading php version, deactivating all other themes and plugins, and nothing worked.

**Comment by zant23** [comment]
I have the same issue on two of my sites. Please fix this

**Comment by amir-zafari** [comment]
I am having the same issue with the Divi theme and PHP 8.2.

**Comment by ngenen** [comment]
I just installed a fresh copy of elementor pro on fresh wordpress 6.6.2 and PHP 8.2. Same issue.

**Comment by bomastudioweb** [comment]
To me, this solved this issue. 

Be sure to check **wp-config.php**

CHANGHE THIS 
define( 'DISALLOW_FILE_EDIT', true );

TO THIS
define( 'DISALLOW_FILE_EDIT', false );

**Comment by jmzolezzi** [comment]
Same issue, using Hello theme.

**Comment by DELTAHighTech** [comment]
Hello,

**Comment by aniaje** [comment]
was the issue solved? i also get the    defined array key "add_ons" in plugins/elementor/modules/home/transformations/filter-plugins.php on line 24. :(

**Comment by JustusRijke** [comment]
Same problem with clean install of wordpress 6.7.2 & elementor 3.27.7, PHP v8.2.12

**Comment by terremoth** [comment]
Same problem here: 

PHP (8.0.30)
WordPress (6.7.1) 
Elementor (3.25.10)

**Comment by k-nowicki** [comment]
Same here.
Clean install

PHP Version:  8.3.20
WP: 6.7.2
Elementor: 3.28.3
Theme: Hello Elementor

Also there is something that might be connected to the issue:
Elementor Library: Not connected (cURL error 28: Resolving timed out after 5002 milliseconds)

**Comment by terremoth** [comment]
[SOLVED] - in my case

I discovered what was the problem in my case. Seems like the Elementor make http requests behind the scenes to get information about its addons, but if your server blocks it (firewall or network capabilities for eg.) it won't be able to reach the addons endpoints, and cause this bugs. I just asked the VPS networks admins to unblock elementor and wordpress URLs from firewall, they did and solved.

**Comment by nicholaszein** [comment]
Hello everyone!

It's been a while sine this issue was reported... 

Because there was no Elementor System Info included in the Bug Report, we couldn't validate this issue.

We have tested various configurations, and could not reproduce this issue. However, @terremoth's [solution](https://github.com/elementor/elementor/issues/27697#issuecomment-2804812979) may be worth testing.

For now we are closing this issue.

If you think this is still an issue caused by Elementor, feel free to open a new 🐞 [Bug Report](https://elemn.to/gh-new-bug-report) following our [Contribution Guidelines](https://elemn.to/gh-contributing-bug-reports) fur submitting new Bug Reports, and we will be happy to verify the issue again.


Kind regards
